Argo/Types/JSON.swift

```diff
@@ -2,7 +2,7 @@ import Foundation
 
 /// A type safe representation of JSON.
 public enum JSON {
-  case object([String: JSON])
+  case object([AnyHashable: JSON])
```

Comment:
I don't think this is necessary as JSON objects must have `String` keys. `[String : Any` should work fine (as it does in my port).